lib/models/food.js

```diff
@@ -27,6 +27,34 @@ function create(name, calories) {
     });
   })
 }
+function find (id) {
+  return database.raw('SELECT * FROM foods WHERE id = ?', id)
+};
+
+function update(req) {
```

Comment:
You did a great job starting this and getting the base update working. The only thing we needed were these chained promises to handle every permutation of update.